.github/workflows/wheels.yml

```diff
@@ -145,7 +145,7 @@ jobs:
     runs-on: macos-latest
     strategy:
       matrix:
-        python: [3.6, 3.7, 3.8]
+        python: [3.7, 3.8]
```

Comment:
3.9 was removed when we had issues with it - I think it can be reinstated now?

Reply:
Good point. Just added a commit to reinstate, seems fine.